File: exercism/c/resistor-color/src/resistor_color.c
#include "resistor_color.h"

int color_code(resistor_band_t resistor_color){
    return resistor_color;
}

static resistor_band_t expected[] = {
    BLACK, BROWN, RED, ORANGE, YELLOW,
    GREEN, BLUE, VIOLET, GREY, WHITE
};


resistor_band_t * colors(){
   return expected;
}

File: exercism/kotlin/resistor-color/src/main/kotlin/ResistorColor.kt
object ResistorColor {

    enum class ResistorColorValue = black, brown, red, orange, yellow, green, blue, violet, grey, white

    fun colorCode(input: String): Int = ResistorColorValue.valueOf(input).ordinal

    fun colors(): List<String> = ResistorColorValue.values().map { it.name }.toList()
}
